### ISSUE TITLE
Topic: Demo Puppeteer and Cypress for E2E + A11y Testing

### ISSUE THREAD

**Issue by s-laugh** [esdc-devcop/esdc-devcop.github.io#85]
### Goals / Recommendation

Show how Puppeteer and Cypress can be leveraged to easily run End-to-end testing and Accessibility testing.

### Community Expectations

Assuming most know almost nothing about Puppeteer or Cypress
Assuming most know they should be doing A11y testing but don't
Assuming most don't consider e2e testing.

### References

https://github.com/esdc-devx/auto-a11y-demo
Relates to #6 #8

**Comment by joel-ld** [comment]
That topic would be very useful. In a world that we want to deploy each day, we have to automate these series of tests.

**Comment by shawnthompson** [comment]
YES!

**Comment by shawnthompson** [comment]
There's also this by CDS

https://github.com/cds-snc/a11y-reporting
